examples/existing-image/README.md

```diff
@@ -1,109 +1,54 @@
-
-### Inputs For This Example
-
-| Name | Description | Type | Example Value |
-|------|-------------|------| ------------- |
-| `acm_certificate_arn` | ACM certificate ARN to use with load balancer | string | `arn:aws:acm:us-east-2:123456:certificate/123abc`
-| `domain_name` | Name of existing DNS Zone in which a record set will be created | string | `example.com` |
-| `friendly_name_prefix` | Name prefix used for resources | string | `somename` |
-| `tfe_subdomain` | Desired DNS record subdomain | string | `tfe` |
-| `tfe_license_name` | The name to use when copying the TFE license file to the EC2 instance. | string | `license.rli` |
-| `tfe_license_filepath` | The absolute path to the TFE license file on the system running Terraform. | string | `Users/yourname/license.rli` |
-| `iact_subnet_list` | A list of CIDR masks that configure the ability to retrieve the IACT from outside the host. | list(string) | `["0.0.0.0/0"]` |
-| `load_balancing_scheme` | Load Balancing Scheme. Supported values are: "PRIVATE"; "PRIVATE_TCP"; "PUBLIC". | string | `PUBLIC` |
-| `ami_id` | AMI ID of the custom image to use for TFE instances. If this value is provided, you do not need any of the following ami variable values. | string | `ami-12345` |
-| `ami_owners` | List of AMI owners to limit search. (Not needed if providing ami_id value.) | list(string) | `["self"]` |
-| `ami_filter_name` | The name of a key off of which to filter with a key/value pair. (Not needed if providing ami_id value.) | string | `"tag:Distro"` |
-| `ami_filter_value` | The value off of which to filter with a key/value pair. (Not needed if providing ami_id value.) | string | `"Ubuntu"` |
-| `ami_most_recent` | If more than one result is returned, use the most recent AMI. (Not needed if providing ami_id value.) | bool | `true` |
-
-### ami_id
-
-The base image used for the custom image should be Ubuntu or RHEL to work with the root
-module as-is.
-
-This example will either use the `ami_id` directly, or you may use a data source to filter
-on the specific AMI to use.
-
-In the `ami_id` data source, you will notice that this example filters on three criteria, a
-unique key/value pair, the virtualization type, and whether or not to use the latest image
-in which this search results. Because it is important that Terraform is only able to find
-one AMI based on the search of this [data source](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/ami),
-you may decide to add more filters in order to narrow down your search.
-
-Otherwise, you may decide to provide the `ami_id` variable directly, instead of using the
-data source. To do this, simply provide a value for the `ami_id` variable with the specific
-AMI ID that you wish to use. If you choose to do this, you do not need to provide values for
-the other variables that begin with `ami_`.
```

Comment:
I think this information is still appropriate to leave in.

Reply:
I will add as a prerequisite